src/olympia/reviews/views.py

```diff
@@ -166,7 +166,11 @@ def delete(request, addon, review_id):
     review = get_object_or_404(Review.objects, pk=review_id, addon=addon)
     if not user_can_delete_review(request, review):
         raise PermissionDenied
-    review.delete(user_responsible=request.user)
+    # Log deleting moderated reviews
+    if review.editorreview:
+        review.moderator_delete(user=request.user)
```

Comment:
The API uses `CanDeleteReviewPermission` which uses `user_can_delete_review` that your patch changes, but it does not uses that view, so it won't do the `moderator_delete()` call, which is untested in the test you link to